optaplanner-wb-webapp/pom.xml

```diff
@@ -825,6 +824,7 @@
     <dependency>
       <groupId>org.jboss.errai</groupId>
       <artifactId>errai-cdi-client</artifactId>
+      <!-- Unlike all other client only dependencies, errai-cdi-client must not be in scope provided -->
```

Comment:
@csadilek Any reason why errai-cdi-client needs to be on the server too?
For UF, kie-wb, etc the naming convention is that client is client side only (and therefore provided scope here).

Setting it as provided gives this exception during deployment:

    [org.jboss.weld.Bootstrap] WELD-000119: Not generating any bean definitions from org.jboss.errai.cdi.server.events.EventDispatcher because of underlying class loading error: Type org.jboss.errai.enterprise.client.cdi.CDIProtocol from [Module "deployment.ROOT.war:main" from Service Module Loader] not found.

And another one later on (caused by this one) that actually kills the deployment.